### DISCUSSION TITLE
Simplyfy and unify JSON structure and remove HTML tags

### DISCUSSION THREAD

**Discussion by tristcoil** [Jitendex/Jitendex#99] [Ideas]
Hello, is there a version of jitendex that has much simpler structure? I tried to parse jitendex and not sure if 1000 line JSON for each word is really way to go going forward. Also I have noticed that there are at least 2 different JSON structures. I can even pay for the simplified JSON version if need be. Thanks, lead dev of hanabira.org

**Comment by stephenmk** [comment]
Jitendex data is intended to be viewed through a dictionary client by end-users. The aim of the project isn't to produce useful data to be analyzed programmatically by other app developers, so consistency and simplicity in that regard is not a concern. Most of the information in Jitendex can be obtained from the [EDRDG](https://www.edrdg.org/wiki/index.php/JMdict-EDICT_Dictionary_Project) in a format that is properly structured to be usable by app developers.

**Reply by tristcoil** [reply]
Thank you for your reply. In the meantime I have found rather useful python wrapper around JMDict https://github.com/neocl/jamdict
